sgx-tools/utils_linux.go

```diff
@@ -17,3 +19,31 @@ func IsProductEnclave(reportBody intelsgx.ReportBody) (bool, error) {
 
 	return false, nil
 }
+
+func readFile(file string, size int64) ([]byte, error) {
```

Comment:
Rename to readAndCheckFile() to reflect the fact that the length of file is always checked.